components/list/demo/list-item-sample.js

```diff
@@ -0,0 +1,47 @@
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { ListItemCheckboxMixin } from '../list-item-checkbox-mixin.js';
+//import { ListItemDragMixin } from '../list-item-drag-mixin.js';
+
+class ListItemSample extends ListItemCheckboxMixin(LitElement) {
+	// TODO: Role and breakpoints to live elsewhere
+	static get properties() {
+		return {
+			href: { type: String }
+		};
+	}
+
+	// TODO: Placeholder grid styles to be replaced with list-item-generic
+	static get styles() {
+		return [ (super.styles ? super.styles : css``), css`
+			.list-grid {
+				display: grid;
+				grid-template-columns:
+					[start outside-control-start] 40px
+					[control-start outside-control-end] 40px
+					[control-end content-start] auto
+					[content-end actions-start] auto
+					[end actions-end];
+			}
+			.d2l-list-item-content {
+				grid-column: content-start / content-end;
+				grid-row: 1 / 2;
+			}
+			.d2l-list-item-actions {
+				grid-column: actions-start / actions-end;
+				grid-row: 1 / 2;
+			}
+		`];
+	}
+	render() {
+		return html`
+			<div class="list-grid">
+				${ this._renderDragHandle ? this._renderDragHandle() : '' }
```

Comment:
Seems like this is future-looking for drag & drop...?  It seems like this conditional should not be needed - this helper would be provided by one of the mixins that are being extended, and we should not need to check for it.

Reply:
Correct, the conditional is meant to be removed once implemented. This is only meant as a hint for developers.